Demo/Common/IntegrationExamples/DropIn/DropInAdvancedFlowExample.swift

```diff
@@ -80,6 +80,7 @@ internal final class DropInAdvancedFlowExample: InitialDataAdvancedFlowProtocol
             } else if let order = response.order,
                       let remainingAmount = order.remainingAmount,
                       remainingAmount.value > 0 {
+                ConfigurationConstants.current.value = remainingAmount.value
```

Comment:
This will update the values in the UserDefaults right?

Reply:
Good catch, I will do this another way.